README.markdown

```diff
@@ -125,7 +119,14 @@ export NVM_DIR="$HOME/.nvm" && (
 ) && . "$NVM_DIR/nvm.sh"
 ```
 
-Add these lines to your `~/.bashrc`, `~/.profile`, or `~/.zshrc` file to have it automatically sourced upon login:
+Or if you have `git` installed (requires git v1.7+):
+
+1. clone this repo in the root of your user profile
+  - `cd ~/` from anywhere then `git clone https://github.com/creationix/nvm.git`
```

Comment:
`git clone https://github.com/creationix/nvm.git .nvm` so it names the folder properly